app/services/stats/calculations.rb

```diff
@@ -100,15 +109,21 @@ def self.calculate_trigger(row)
     end
 
     def self.calculate_bu_non_trigger(row)
-      calculate_percentage(row[:non_trigger_responded_in_time], row[:non_trigger_responded_in_time] + row[:non_trigger_responded_late] + row[:non_trigger_open_late])
+      calculate_percentage(
+        row[:non_trigger_responded_in_time] + row[:non_trigger_open_in_time], 
+        row[:non_trigger_responded_in_time] + row[:non_trigger_responded_late] + row[:non_trigger_open_late] + row[:non_trigger_open_in_time])
     end
 
     def self.calculate_bu_trigger(row)
-      calculate_percentage(row[:trigger_responded_in_time], row[:trigger_responded_in_time] + row[:trigger_responded_late] + row[:trigger_open_late])
+      calculate_percentage(
```

Comment:
Really small thing but it might be good to have these additions stored in a variable and the fed into the `calculate_percentage` function like is done in `calculate_bu_overall_performance`

Up to you though not a big thing 👍